### DISCUSSION TITLE
can i make it so otd opens when I start up my pc?

### DISCUSSION THREAD

**Discussion by ghost** [OpenTabletDriver/OpenTabletDriver#1440] [Q&A] [answered]
can I do that?

**Comment by jamesbt365** [comment]
- Right click OpenTabletDriver.UX.Wpf.exe (should be in %localappdata%\OpenTabletDriver\bin when installer is used) -> Create shortcut
- Right click shortcut -> Properties -> Run -> Minimized
- Press Win + R -> Type shell:startup -> Press Enter
- Move the shortcut to the folder that opened

**Reply by ghost** [reply]
tysm!

**Comment by gonX** [comment, marked as answer]
Now that our wiki is updated, I think it is more appropriate to link to the guides instead:

- [Windows](https://opentabletdriver.net/Wiki/FAQ/Windows#startup)
- [Linux](https://opentabletdriver.net/Wiki/FAQ/Linux#autostart)

nothing for MacOS at the moment
